### PR TITLE
Fix formatting, and use better css minification class

### PR DESCRIPTION
**Pull request by GrahamCampbell**
There were some occasions where tabs creeped in where you have been mainly indenting with 4 spaces. I have made sure 4 spaces are used in all cases. I have also fixed 2 or 3 other small errors and changed the css minification class to a better one from the minification library.

### CODE REVIEW

File: src/Lightgear/Asset/Asset.php [hunk 1]
```diff
@@ -413,8 +413,7 @@ protected function publishAsset($asset, $type)
     protected function minifyAsset($contents, $type)
     {
         if ($type === 'styles') {
-            $cssMin = new \CSSmin;
-            return $cssMin->run($contents);
+            return \Minify_CSS::minify($contents, array('preserveComments' => false));
```
Comment (ralf57) [inline src/Lightgear/Asset/Asset.php:416]: Are you sure this one performs better?

**Comment by GrahamCampbell** [comment]
Not 100% certain, but it seems more widely used. Anyway, in production, we would be caching the assets, so if the minification is better, but the performance is slightly lower for the initial generation, I'd say that's the right way round.

**Comment by ralf57** [comment]
Merged, thanks.
FYI, I am doing a major refactoring on the "develop" branch.

**Comment by GrahamCampbell** [comment]
Ok. Thanks.

**Pull request merged**